test/unit/resources/postgres_session_test.rb

```diff
@@ -11,4 +11,8 @@
     resource = load_resource("postgres_session", "myuser", "mypass", "127.0.0.1")
     _(resource.send(:create_psql_cmd, "SELECT current_setting('client_min_messages')", ["testdb"])).must_equal "PGPASSWORD='mypass' psql -U myuser -d testdb -h 127.0.0.1 -A -t -c SELECT\\ current_setting\\(\\'client_min_messages\\'\\)"
   end
+  it "verify postgres_session create_psql_cmd using sensitive argument" do
+    resource = load_resource("postgres_session", "myuser", "mypass", "127.0.0.1", sensitive: true)
+    _(resource.send(:create_psql_cmd, "SELECT * FROM STUDENTS;", ["testdb"])).must_equal "PGPASSWORD='REDACTED' psql -U myuser -d testdb -h 127.0.0.1 -A -t -c SELECT\\ \\*\\ FROM\\ STUDENTS\\;"
```

Comment:
For some reason the spec isn't quite redacting here 🤔 

```

1) Failure:
--
  | Inspec::Resources::PostgresSession#test_0003_verify postgres_session create_psql_cmd using sensitive argument [/workdir/test/unit/resources/postgres_session_test.rb:16]:
  | expected | 0s
  | actual | 1s
  | @@ -1 +1 @@
  | -"PGPASSWORD='REDACTED' psql -U myuser -d testdb -h 127.0.0.1 -A -t -c SELECT\\ \\*\\ FROM\\ STUDENTS\\;"
  | +"PGPASSWORD='mypass' psql -U myuser -d testdb -h 127.0.0.1 -A -t -c SELECT\\ \\*\\ FROM\\ STUDENTS\\;"
  |  
```

Reply:
Yeah, I tested it using https://rubular.com/, but seems like I still need to change something there.

Reply:
![Screenshot 2020-07-01 at 12 42 10](https://user-images.githubusercontent.com/7865030/86240051-5d1fa700-bb98-11ea-89ee-bd3466e0803a.png)

This is very interesting.... I just played around on rubular as well and it seems like you're right on here.. 🤔  I wonder if it might be worth actually prying in and seeing why it doesn't like that regex.